aleth/MinerAux.h

```diff
@@ -52,11 +52,8 @@ class MinerCLI
         Benchmark
     };
 
-
     explicit MinerCLI(OperationMode _mode = OperationMode::None): mode(_mode)
     {
-        Ethash::init();
-        NoProof::init();
         BasicAuthority::init();
```

Comment:
We probably should also drop `BasicAuthority::init()`.

Reply:
Ok, I'll remove it too. By the way, I just found these `init` in `WebThreeDirect::WebThreeDirect` too. `WebThreeDirect` is initialized in `main` at line 854 so I'm pretty sure those init are redundant too, but I need your confirmation.

Reply:
[The inits in `WebThreeDirect` ctor](https://github.com/ethereum/aleth/blob/master/libwebthree/WebThree.cpp#L49)

Reply:
@gumb0 can you look here. Should WebTreeDirect init seal engines? Do we need to init BasicAuthority engine? It seems to be abandoned.

Reply:
Some unit tests create `WebThreeDirect` instances, but it seems testeth code calls these `init`s anyway for each test. So I think we can remove it from `WebThreeDirect` if tests don't break

Reply:
Seems that there are some unit tests for `BlockChain` that use `BasicAuthority` (`BlockChainInsertTests`)

@winsvega Do you know if `BasicAuthority` is useful for anything?

Reply:
Without this init. Windows was having build errors

Reply:
My theory is that visual studio does not build cpp file if nothing from this file is called.

Reply:
@gumb0 I have removed those inits, from `WebThreeDirect` too. No failed unit tests.

@chfast meant to drop `BasicAuthority::init()` from `MinerCLI` ctor, not the `BasicAuthority` itself. I just checked and see `BasicAuthority` is used in [BlockChainInsert.cpp](https://github.com/ethereum/aleth/blob/master/test/unittests/libethereum/BlockChainInsert.cpp) test.